bxt-p.mk

```diff
@@ -59,13 +61,13 @@ optee_client:
 	@echo '****************************************************************'
 	$(OPTEE_CLIENT_ENV_VAR) $(MAKE) -C optee_client
 
-optee_test:
+optee_test: optee_client optee_os
 	@echo '****************************************************************'
 	@echo '*   build optee_test ...'
 	@echo '****************************************************************'
 	$(OPTEE_TEST_ENV_VAR) $(MAKE) -C optee_test
 
-ikgt:
+ikgt: optee_os
```

Comment:
ikgt doesn't have to depend on optee_os. we can keep it clean.

Reply:
Hi Roger,

ikgt needs to pack op-tee image on this platform. So put such kind of dependency here.